examples/dt/uni02beta/control-plane/service-values.yaml

```diff
@@ -0,0 +1,115 @@
+---
+apiVersion: v1
+kind: ConfigMap
+
+metadata:
+  name: service-values
+  annotations:
+    config.kubernetes.io/local-config: "true"
+
+data:
+  cinder:
+    extraMounts:
```

Comment:
The extraMounts should be deleted because the shares-config file is not going to be used in the deployment.